pkg/importer/upload-datasource.go

```diff
@@ -51,8 +54,7 @@ func (ud *UploadDataSource) Info() (ProcessingPhase, error) {
 	if ud.contentType == cdiv1.DataVolumeArchive {
 		return ProcessingPhaseTransferDataDir, nil
 	}
-	if !ud.readers.Convert {
-		// Uploading a raw file, we can write that directly to the target.
+	if ud.directWriteException {
```

Comment:
Maybe pass the source content type and do the `sourceContentType == common.BlockdeviceClone` here? It makes this easier to debug and understand than having an arbitrary `directWriteException` that's not too informative. Just a nit though, feel free to ignore.

Reply:
yeah that makes sense